File: retrieve.php
<?php
$m->MongoClient();
$db = $m->selectDB('TYITDB1');
$collection = $db->selectCollection('TYITCOL');
// retrieve data
$cursor = $collection->find();
foreach ($cursor as $document) {
    echo $document["name"] . "\n";
}
?>

File: retrieve.py
from pymongo import MongoClient
client = MongoClient('localhost',27017)
db = client.TYITDB1
collection = db.TYITCOL
def retrieve():
    try:
        for studentinfo in collection.find():
            print(studentinfo)
    except Exception as e:
        print(str(e))
retrieve()